src/index.html

```diff
@@ -5,8 +5,315 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__block">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">smartphones</a></li>
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover">
+              laptops & computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="nav-hover"
            >
              laptops & computers
            </a>
```
Closing tag should be just under the opening

Reply:
CID :)